### ISSUE TITLE
Feedback to SCI for AI Specification — extending the functional unit to water consumption

### ISSUE THREAD

**Issue by clauramirezdauria** [Green-Software-Foundation/sci-ai#152]
<html>
<body>
<p dir="ltr"><strong>Type:</strong> Feedback / scope proposal
<strong>Refers to:</strong> SPEC.md — functional unit (R), Consumer boundary, Operation and Monitoring lifecycle stage</p>

<hr>
<h2 dir="ltr">Summary</h2>

<p dir="ltr">The specification already solves the hard part of per-service environmental accounting: it defines a <strong>functional unit R</strong> and enumerates concrete units for the Consumer boundary — per token, per image, per second of video, per second of audio processed, per page processed.</p>

<p dir="ltr">Carbon is measured against that denominator. <strong>Water is not measured at all.</strong></p>

<p dir="ltr">This is not a criticism of scope discipline — SCI is a carbon standard and it says so. The proposal is narrower and, I think, cheap: <strong>the denominator that SCI for AI has already defined and ratified is the missing piece for water accounting too, and the numerator already exists in a published standard and in a working open-source implementation.</strong> No new research is required to close this gap; only a decision to reuse <code>R</code>.</p>

<hr>
<h2 dir="ltr">1. Why water, and why now</h2>

<p dir="ltr">Water is the impact category where regulation is currently moving fastest, and where the per-service layer is missing everywhere:</p>

<ul dir="ltr">
<li><strong>EU AI Act, Annex XI</strong> requires GPAI providers to document <strong>energy</strong> consumption. Water is absent.</li>
<li><strong>Data-centre-level water regulation is arriving.</strong> Spain's draft Royal Decree on data centre sustainability (public consultation closed 4 September 2026) caps <strong>WUE at 0.1 L/kWh</strong> and PUE at 1.15. Brazil's REDATA bill (PL 278/2026, in the Senate) proposes a water efficiency index of <strong>0.05 L/kWh</strong>. The EU data centre rating scheme is built on PUE and WUE.</li>
<li><strong>All of it stops at the facility.</strong> Every instrument above measures a building. None of them can answer what a single inference cost, which is the only unit a user or a procurement team can act on.</li>
</ul>

<p dir="ltr">That per-service layer is exactly where SCI for AI already operates, and where no other instrument does.</p>

<p dir="ltr">The timing is also practical: the European Commission's targeted consultation on <em>measuring energy consumption and emissions of AI models and systems</em> closed on 25 May 2026, with a stated aim of producing a measurement framework and a possible <strong>AI energy and emissions label</strong>; the Commission must report on harmonised standards by <strong>2 August 2028</strong>. A standard that covers water by then is referenceable. One that does not, is not.</p>

<p dir="ltr">The GSF's own policy research on SCI for AI and EU AI Act compliance already names this: it notes that future standards <em>could</em> extend to <strong>water consumption (data centre cooling)</strong> and e-waste. This issue is a request to move that from "could" to a scoped work item.</p>

<hr>
<h2 dir="ltr">2. The numerator already exists — twice</h2>

<p dir="ltr"><strong>a) In a published standard.</strong> The Green Grid's White Paper #35, which originated the WUE metric, does not define one indicator but a pair:</p>

<div role="group" aria-label="Código" tabindex="0"><div><div></div></div><div><pre style="color: rgb(20, 24, 31); background: transparent; font-family: var(--font-mono);"><code style="color: rgb(20, 24, 31); background: transparent; font-family: var(--font-mono); white-space: pre-wrap;">WUE         = annual site water usage / IT equipment energy
WUE_source  = [ EWIF × PUE ] + annual site water usage / IT equipment energy</code></pre></div></div>

<p dir="ltr">where <strong>EWIF</strong> (Energy Water Intensity Factor) is the water consumed off-site in generating the electricity used on-site, in L/kWh. WP#35 defines <code>WUE_source</code> as covering <em>"water used on-site and water used off-site in the production of the energy used on-site"</em>, and supplies a default value (1.8 L/kWh, a US national average from NREL 2003 / US DOE 2006) plus regional tables, for use when local data is unavailable.</p>

<p dir="ltr"><strong>b) In a working open-source implementation.</strong> EcoLogits computes a water consumption footprint <strong>per request</strong> as:</p>

<div role="group" aria-label="Código" tabindex="0"><div><div></div></div><div><pre style="color: rgb(20, 24, 31); background: transparent; font-family: var(--font-mono);"><code style="color: rgb(20, 24, 31); background: transparent; font-family: var(--font-mono); white-space: pre-wrap;">WCF_request = E_server × [ WUE_on-site + PUE × WUE_off-site ]</code></pre></div></div>

<p dir="ltr">sourcing the off-site term from <strong>World Resources Institute</strong> methodology, falling back to a world default <strong>with an explicit warning emitted</strong> when country-level data is unavailable.</p>

<p dir="ltr">These are structurally the same calculation: <code>WUE_off-site</code> occupies the position of <code>EWIF</code>. <strong>The method is settled and independently implemented. What is missing is a standard that expresses it against a common functional unit.</strong></p>

<hr>
<h2 dir="ltr">3. Why the site term alone is not sufficient</h2>

<p dir="ltr">Worth stating explicitly, because facility-level regulation is converging on the site term only.</p>

<p dir="ltr">Applying WP#35's formula to Spain's proposed caps (PUE 1.15, WUE 0.1) with WP#35's own default EWIF of 1.8 L/kWh:</p>

<div dir="ltr">
Term | Calculation | L/kWh | Share
-- | -- | -- | --
Generation (EWIF × PUE) | 1.8 × 1.15 | 2.07 | 95.4 %
Site (WUE) | — | 0.10 | 4.6 %
Total (WUE_source) | 2.07 + 0.10 | 2.17 | 100 %

</div>

<p dir="ltr"><strong>Caveat, stated plainly:</strong> 1.8 L/kWh is a US average from 2003–2006 and is not applicable to Spain or to any specific grid; it is used here only because it is WP#35's own default. The point is not the magnitude but the <strong>structure</strong> — a site-only indicator cannot see the generation term at all, whatever its local value.</p>

<p dir="ltr">This has a consequence that matters for the standard's usefulness: because the generation term depends on the generation mix, <strong>two AI workloads with identical site WUE but different grids have very different water footprints, and a site-only metric reports them as equal.</strong> Conversely, a workload's shift to low-water generation is invisible. Carbon accounting solved this problem years ago with regional carbon intensity (<code>I</code> in the SCI formula). Water has the same shape and no equivalent.</p>

<hr>
<h2 dir="ltr">4. Concrete proposal</h2>

<p dir="ltr">Minimal version, in order of increasing ambition — I would advocate for (1):</p>

<ol dir="ltr">
<li><strong>A companion water-intensity metric reusing <code>R</code> unchanged.</strong> Same functional units already ratified (per token, per image, per second), same boundaries, same lifecycle stages. Numerator per §2. This could live as an annex to SPEC.md or as a parallel short specification; it requires no change to the carbon methodology.</li>
<li><strong>A published guidance note</strong> on the on-site / off-site split and on acceptable sources for the off-site factor (WRI, national grid operators, EWIF regional tables), so implementations are comparable rather than each choosing its own.</li>
<li><strong>Adopt EcoLogits' defaulting convention.</strong> When a regional factor is unavailable and a global default is substituted, the output should carry an explicit uncertainty flag. This is a small design rule with a large effect on whether published figures can be compared or aggregated, and it is worth writing into the standard rather than leaving to implementers.</li>
</ol>

<p dir="ltr">If the working group's view is that water belongs outside SCI entirely, that is a legitimate scoping decision — but it would be valuable to say so explicitly in the specification's scope section, so that implementers and regulators know the gap is deliberate rather than pending.</p>

<hr>
<h2 dir="ltr">5. Disclosure and what I can contribute</h2>

<p dir="ltr">I am an independent consultant and postgraduate researcher based in Argentina, working on financial and environmental cost transparency of AI use for small and medium enterprises, in Spanish-speaking markets. I have no commercial interest in data centre infrastructure or in any AI vendor, and I am not a GSF member.</p>

<p dir="ltr">I do not bring primary measurement data on water coefficients, and I would rather say so than imply otherwise.</p>

<p dir="ltr">What I can contribute, if it is useful to the group:</p>

<ul dir="ltr">
<li><strong>Empirical data on credit-based and subscription platforms</strong> (automation and agent platforms billing in opaque "credits" rather than tokens), where the functional unit is deliberately hidden from the user. This is a measurement blind spot that sits outside every API-instrumented tool I have surveyed, including EcoLogits and AI Energy Score, and it is where most SME and non-technical usage actually happens. I have run controlled repeated-execution measurements on such platforms.</li>
<li><strong>Review and Spanish-language translation</strong> of any water annex, and a bridge to the Spanish and Latin American regulatory processes cited above, where per-service metrics are currently absent from the debate.</li>
</ul>

<p dir="ltr">Happy to be redirected if this belongs in a different working group or repository.</p>

<hr>
<h2 dir="ltr">References</h2>

<ul dir="ltr">
<li>Green Software Foundation, <em>Software Carbon Intensity for AI and EU AI Act Environmental Compliance</em> — <a href="https://greensoftware.foundation/policy/research/sci-ai-eu-ai-act/">https://greensoftware.foundation/policy/research/sci-ai-eu-ai-act/</a></li>
<li>The Green Grid, White Paper #35, <em>Water Usage Effectiveness (WUE): A Green Grid Data Center Sustainability Metric</em> — <a href="https://www.thegreengrid.org/en/resources/library-and-tools/238-WP%2335---Water-Usage-Effectiveness-(WUE):-A-Green-Grid-Data-Center-Sustainability-Metric">https://www.thegreengrid.org/en/resources/library-and-tools/238-WP%2335---Water-Usage-Effectiveness-(WUE):-A-Green-Grid-Data-Center-Sustainability-Metric</a></li>
<li>EcoLogits, <em>Methodology — LLM Inference</em> (water consumption footprint) — <a href="https://ecologits.ai/latest/methodology/llm_inference/">https://ecologits.ai/latest/methodology/llm_inference/</a></li>
<li>Commission Delegated Regulation (EU) 2024/1364 (EU data centre common rating scheme; PUE and WUE) — <a href="https://www.boe.es/buscar/doc.php?id=DOUE-L-2024-80715">https://www.boe.es/buscar/doc.php?id=DOUE-L-2024-80715</a></li>
<li>MITECO (Spain), draft Royal Decree on data centre sustainability requirements, public consultation 27 Aug – 4 Sep 2026 — <a href="https://www.miteco.gob.es/es/energia/participacion/2026/detalle-participacion-publica-k-851.html">https://www.miteco.gob.es/es/energia/participacion/2026/detalle-participacion-publica-k-851.html</a></li>
<li>European Commission, <em>Targeted consultation on measuring energy consumption and emissions of AI models and systems</em> — <a href="https://digital-strategy.ec.europa.eu/en/consultations/targeted-consultation-measuring-energy-consumption-and-emissions-ai-models-and-systems">https://digital-strategy.ec.europa.eu/en/consultations</a></li></ul>
</body>
</html>Type: Feedback / scope proposal Refers to: SPEC.md — functional unit (R), Consumer boundary, Operation and Monitoring lifecycle stage

Summary

The specification already solves the hard part of per-service environmental accounting: it defines a functional unit R and enumerates concrete units for the Consumer boundary — per token, per image, per second of video, per second of audio processed, per page processed.

Carbon is measured against that denominator. Water is not measured at all.

This is not a criticism of scope discipline — SCI is a carbon standard and it says so. The proposal is narrower and, I think, cheap: the denominator that SCI for AI has already defined and ratified is the missing piece for water accounting too, and the numerator already exists in a published standard and in a working open-source implementation. No new research is required to close this gap; only a decision to reuse R.

1. Why water, and why now

Water is the impact category where regulation is currently moving fastest, and where the per-service layer is missing everywhere:

EU AI Act, Annex XI requires GPAI providers to document energy consumption. Water is absent.
Data-centre-level water regulation is arriving. Spain's draft Royal Decree on data centre sustainability (public consultation closed 4 September 2026) caps WUE at 0.1 L/kWh and PUE at 1.15. Brazil's REDATA bill (PL 278/2026, in the Senate) proposes a water efficiency index of 0.05 L/kWh. The EU data centre rating scheme is built on PUE and WUE.
All of it stops at the facility. Every instrument above measures a building. None of them can answer what a single inference cost, which is the only unit a user or a procurement team can act on.

That per-service layer is exactly where SCI for AI already operates, and where no other instrument does.

The timing is also practical: the European Commission's targeted consultation on measuring energy consumption and emissions of AI models and systems closed on 25 May 2026, with a stated aim of producing a measurement framework and a possible AI energy and emissions label; the Commission must report on harmonised standards by 2 August 2028. A standard that covers water by then is referenceable. One that does not, is not.

The GSF's own policy research on SCI for AI and EU AI Act compliance already names this: it notes that future standards could extend to water consumption (data centre cooling) and e-waste. This issue is a request to move that from "could" to a scoped work item.

2. The numerator already exists — twice

a) In a published standard. The Green Grid's White Paper #35, which originated the WUE metric, does not define one indicator but a pair:

WUE         = annual site water usage / IT equipment energy
WUE_source  = [ EWIF × PUE ] + annual site water usage / IT equipment energy

where EWIF (Energy Water Intensity Factor) is the water consumed off-site in generating the electricity used on-site, in L/kWh. WP#35 defines WUE_source as covering "water used on-site and water used off-site in the production of the energy used on-site", and supplies a default value (1.8 L/kWh, a US national average from NREL 2003 / US DOE 2006) plus regional tables, for use when local data is unavailable.

b) In a working open-source implementation. EcoLogits computes a water consumption footprint per request as:

WCF_request = E_server × [ WUE_on-site + PUE × WUE_off-site ]

sourcing the off-site term from World Resources Institute methodology, falling back to a world default with an explicit warning emitted when country-level data is unavailable.

These are structurally the same calculation: WUE_off-site occupies the position of EWIF. The method is settled and independently implemented. What is missing is a standard that expresses it against a common functional unit.

3. Why the site term alone is not sufficient

Worth stating explicitly, because facility-level regulation is converging on the site term only.

Applying WP#35's formula to Spain's proposed caps (PUE 1.15, WUE 0.1) with WP#35's own default EWIF of 1.8 L/kWh:

Term	Calculation	L/kWh	Share
Generation (EWIF × PUE)	1.8 × 1.15	2.07	95.4 %
Site (WUE)	—	0.10	4.6 %
Total (WUE_source)	2.07 + 0.10	2.17	100 %

Caveat, stated plainly: 1.8 L/kWh is a US average from 2003–2006 and is not applicable to Spain or to any specific grid; it is used here only because it is WP#35's own default. The point is not the magnitude but the structure — a site-only indicator cannot see the generation term at all, whatever its local value.

This has a consequence that matters for the standard's usefulness: because the generation term depends on the generation mix, two AI workloads with identical site WUE but different grids have very different water footprints, and a site-only metric reports them as equal. Conversely, a workload's shift to low-water generation is invisible. Carbon accounting solved this problem years ago with regional carbon intensity (I in the SCI formula). Water has the same shape and no equivalent.

4. Concrete proposal

Minimal version, in order of increasing ambition — I would advocate for (1):

A companion water-intensity metric reusing R unchanged. Same functional units already ratified (per token, per image, per second), same boundaries, same lifecycle stages. Numerator per §2. This could live as an annex to SPEC.md or as a parallel short specification; it requires no change to the carbon methodology.
A published guidance note on the on-site / off-site split and on acceptable sources for the off-site factor (WRI, national grid operators, EWIF regional tables), so implementations are comparable rather than each choosing its own.
Adopt EcoLogits' defaulting convention. When a regional factor is unavailable and a global default is substituted, the output should carry an explicit uncertainty flag. This is a small design rule with a large effect on whether published figures can be compared or aggregated, and it is worth writing into the standard rather than leaving to implementers.

If the working group's view is that water belongs outside SCI entirely, that is a legitimate scoping decision — but it would be valuable to say so explicitly in the specification's scope section, so that implementers and regulators know the gap is deliberate rather than pending.

5. Disclosure and what I can contribute

I am an independent consultant and postgraduate researcher based in Argentina, working on financial and environmental cost transparency of AI use for small and medium enterprises, in Spanish-speaking markets. I have no commercial interest in data centre infrastructure or in any AI vendor, and I am not a GSF member.

I do not bring primary measurement data on water coefficients, and I would rather say so than imply otherwise.

What I can contribute, if it is useful to the group:

Empirical data on credit-based and subscription platforms (automation and agent platforms billing in opaque "credits" rather than tokens), where the functional unit is deliberately hidden from the user. This is a measurement blind spot that sits outside every API-instrumented tool I have surveyed, including EcoLogits and AI Energy Score, and it is where most SME and non-technical usage actually happens. I have run controlled repeated-execution measurements on such platforms.
Review and Spanish-language translation of any water annex, and a bridge to the Spanish and Latin American regulatory processes cited above, where per-service metrics are currently absent from the debate.

Happy to be redirected if this belongs in a different working group or repository.

References
Green Software Foundation, Software Carbon Intensity for AI and EU AI Act Environmental Compliance — https://greensoftware.foundation/policy/research/sci-ai-eu-ai-act/
The Green Grid, White Paper #35, Water Usage Effectiveness (WUE): A Green Grid Data Center Sustainability Metric — https://www.thegreengrid.org/en/resources/library-and-tools/238-WP%2335---Water-Usage-Effectiveness-(WUE):-A-Green-Grid-Data-Center-Sustainability-Metric
EcoLogits, Methodology — LLM Inference (water consumption footprint) — https://ecologits.ai/latest/methodology/llm_inference/
Commission Delegated Regulation (EU) 2024/1364 (EU data centre common rating scheme; PUE and WUE) — https://www.boe.es/buscar/doc.php?id=DOUE-L-2024-80715
MITECO (Spain), draft Royal Decree on data centre sustainability requirements, public consultation 27 Aug – 4 Sep 2026 — https://www.miteco.gob.es/es/energia/participacion/2026/detalle-participacion-publica-k-851.html
European Commission, Targeted consultation on measuring energy consumption and emissions of AI models and systems — [https://digital-strategy.ec.europa.eu/en/consultations](https://digital-strategy.ec.europa.eu/en/consultations/targeted-consultation-measuring-energy-consumption-and-emissions-ai-models-and-systems)

**Comment by clauramirezdauria** [comment]
Cross-reference: this proposal connects directly to point 4 raised by @bgamazay in #128 — "for LLMs I think Query should be the FU rather than Token... The query is really what is desired by the end user."

Supporting evidence for that point, since it strengthens both cases: Bai et al. (arXiv:2604.22750, April 2026 — Stanford / MIT / Microsoft Research) measured token consumption in agentic coding tasks and found that runs of the same task can differ by up to 30x in total tokens, that agentic tasks consume ~1000x more tokens than chat, and that frontier models fail to predict their own token usage (correlations up to 0.39, systematically underestimating).

If token counts for the same requested outcome vary by that margin, the token is a weak functional unit for anything an end user or a procurement team needs to compare — which is the argument in #128.

It is also why a water metric should attach to the same unit as carbon, whichever unit the group settles on: the choice of R is what makes both metrics comparable, or not.
